view/GameView/SettingsScreen.py

```diff
@@ -0,0 +1,170 @@
+        self.manager.process_events(event)
+
+        if event.type == pygame.USEREVENT and event.user_type == pygame_gui.UI_BUTTON_PRESSED:
+            switcher = {
+                self.pause_game_button: self._pause_pressed,
+                self.leave_game_button: self._leave_pressed,
+                self.return_to_game_button: self._return_pressed,
+                self.shortcuts_help_button: self._update_shortcuts_textbox
+            }
+            try:
+                switcher.get(event.ui_element)()
+            except TypeError:
+                logging.warning("Did not find UI-Element in dict")
+
+        if event.type == pygame.KEYDOWN and event.key == pygame.K_ESCAPE:
+            self._return_pressed()
+
+    def _pause_pressed(self):
+        # toggle game paused
+        paused = self.controller.lib_client_handler.lib_client.isGamePaused()
+        ret = self.controller.send_request_game_pause(not paused)
+        logging.info(f"Successfully sent pause_message: {ret}")
+
+    def _leave_pressed(self):
+        ret = self.controller.send_game_leave()
+        logging.info(f"Successfully sent game leave message: {ret}")
+        if ret:
+            self.controller.to_main_menu()
+
+    def _return_pressed(self):
+        self.parent_view.parent.to_playing_field()
+
+    def _update_label(self) -> None:
+        paused = self.controller.lib_client_handler.lib_client.isGamePaused()
+        label_str = "Game is paused" if paused else "Game is not paused"
+        self.pause_state_label.set_text(label_str)
+
+    def _update_shortcuts_textbox(self) -> None:
+        if self.shortcuts_help_textbox is not None:
+            self.shortcuts_help_textbox.kill()
+            self.shortcuts_help_textbox = None
+            return
+
+        self.shortcuts_help_textbox = pygame_gui.elements.UITextBox(
+            html_text=self._shortcuts_help_str,
+            relative_rect=pygame.Rect(self.__shortcuts_position, self.__shortcuts_size),
+            manager=self.manager,
+            object_id="#shortcut_textbox"
+        )
+
+    def _init_ui_elements(self):
+        self.shortcuts_help_textbox = None
+
+        self.pause_state_label = pygame_gui.elements.UILabel(
+            relative_rect=pygame.Rect((0,
+                                       self.__padding * len(
+                                           self.container.elements)),
+                                      self.__buttonSize),
+            text="",
+            manager=self.manager,
+            container=self.container,
+            object_id="#pause_state_label"
+        )
+
+        self.pause_game_button = pygame_gui.elements.UIButton(
+            relative_rect=pygame.Rect((0,
+                                       self.__padding * len(
+                                           self.container.elements)),
+                                      self.__buttonSize),
+            text=self._text_buttons["pause_game"],
```

Comment:
toggle label
```suggestion
            paused = self.controller.lib_client_handler.lib_client.isGamePaused()
            text=self._text_buttons["pause_game" if not paused else "unpause_game"],
```
AND label might need to be updated when GamePause messages arrives while screen is open